tests.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# Modified version of script from here:
+#   http://stackoverflow.com/questions/26082444/how-to-work-around-travis-cis-4mb-output-limit
```

Comment:
I did a test with suppressed output and it doesn't produce any input unless an error occurs. We'd need to ping Travis or split the integTest task into several tasks that take less than 10 min to run to avoid the output timeout.